task_6.py

```diff
@@ -0,0 +1,35 @@
+import sqlite3
```

Comment:
so, it looks like you solved only a few exercises.
are you going to finish the task?

Reply:
you're right. tomorrow until 15:00 I will send the rest of the tasks